src/org/etools/j1939_84/bus/j1939/J1939TP.java

```diff
@@ -398,7 +395,7 @@ private Packet sendBam(Packet packet) throws BusException {
                              end);
             Arrays.fill(buf, end + 1, buf.length, (byte) 0xFF);
             buf[0] = (byte) (i + 1);
-            sleep(50);
+            sleep();
```

Comment:
Since sleep now means sleepForBAMDT, an not a generic sleep, we should probably change the name of sleep.